src/lightkurve/utils.py

```diff
@@ -871,3 +875,224 @@ def finalize_notebook_url(notebook_url):
+            "rename_out": ("ID", "pmDEC", "Gaia_G_Mag"),
+        },
+    }
+
+    # Check to make sure that user input is in the correct format
+    if not isinstance(coord, SkyCoord):
+        raise TypeError("Must pass an `astropy.coordinates.SkyCoord` object.")
+    if not isinstance(epoch, Time):
+        raise TypeError("Must pass an `astropy.time.Time object`.")
+    if not isinstance(equinox, Time):
+        raise TypeError("Must pass an `astropy.time.Time object`.")
+
+    # Here we check to make sure that the radius entered is in arcseconds
+    # This also means we do not need to specify arcseconds in our catalog query
+    try:
+        radius = u.Quantity(radius, "arcsecond")
+    except u.UnitConversionError:
+        raise
+
+    # Check to make sure that the catalog provided by the user is valid for this function
+    if catalog.lower() not in _Catalog_Dictionary.keys():
+        raise ValueError(f"Can not parse catalog name '{catalog}'")
+
+    # Get the Vizier catalog name
+    catalog_name = _Catalog_Dictionary[catalog.lower()]["catalog"]
+
+    # Get the appropriate column names and filters to be applied
+    filters = Vizier(
+        columns=_Catalog_Dictionary[catalog.lower()]["columns"],
+        column_filters={
+            _Catalog_Dictionary[catalog.lower()][
+                "column_filters"
+            ]: f"<{magnitude_limit}"
+        },
+    )
+
+    # The catalog can cut off at 50 - we dont want this to happen
+    filters.ROW_LIMIT = -1
+    # Now query the catalog
+    result = filters.query_region(coord, catalog=catalog_name, radius=Angle(radius))[
+        catalog_name
+    ]
+
+    # Rename the columns so that the output is uniform
+    result.rename_columns(
+        _Catalog_Dictionary[catalog.lower()]["rename_in"],
+        _Catalog_Dictionary[catalog.lower()]["rename_out"],
+    )
+
+    # Based on the input coordinates pick the object with the mininmum separation as the reference star.
+
+    c1 = SkyCoord(result["RAJ2000"], result["DEJ2000"], unit="deg")
+
+    if target_coord is None:
+        sep = coord.separation(c1)
+    else:
+        sep = target_coord.separation(c1)
+
+    # Find the object with the minimum separation - this is our target
+    ref_index = np.argmin(sep)
+
+    # apply_propermotion
+    result = _apply_propermotion(result, equinox=equinox, epoch=epoch)
+
+    # Now we want to repete but using the values corrected for proper motion
+    # First get the correct values for target
+    coord_pm_correct = SkyCoord(
+        result["RA"][ref_index], result["DEC"][ref_index], unit="deg"
+    )
+
+    c1_pm_correct = SkyCoord(result["RA"], result["DEC"], unit="deg")
+
+    # Then calculate the separation based on pm corrected values
+    sep_pm_correct = coord_pm_correct.separation(c1_pm_correct)
+
+    # Provide the separation in the output table
+    result["Separation"] = sep_pm_correct
+
+    # Calculate the relative flux
+    result["Relative_Flux"] = np.exp(
```

Comment:
I believe the relative flux calculation is buggy. `np.exp()` should probably be replaced by `10 **`. 

E.g., assume the target star is of magnitude 10, the nearby star is of magnitude 9, i.e., the relative flux is about ~2.512.

```
np.exp((9 - 10) / -2.5) : 1.492
10 ** ((9 - 10) / -2.5) : 2.512
```